packages/backend/src/rpc/dispatcher.spec.ts

```diff
@@ -274,13 +276,33 @@ describe('RPCDispatcher', () => {
           res = mockResponse();
         },
       );
+
       // mock middleware call and prepare erroneous response
       app.overrideSingleton(
         RPCMiddlewareContainer,
         class MockRPCMiddlewareContainer extends RPCMiddlewareContainer {
-          call = jest.fn(() => Promise.reject({ isJoi: true, details: [] }));
+          call = jest.fn(() => Promise.reject({ isJoi: true, details: 'Error details' }));
         },
       );
+
+      // register dummy handler to ensure the correct handler can be reached
+      let dummyHandler = jest.fn((_e: Error) => undefined);
+
+      // register custom handler that we expect to be reached
```

Comment:
The comments here can be changed to start with `create` instead of `register`.

Reply:
will do